apps/src/code-studio/pd/workshop_dashboard/components/workshop_table.jsx

```diff
@@ -2,116 +2,228 @@
  * Table displaying workshop summaries based on a supplied query.
  */
 
-import $ from 'jquery';
-import _ from 'lodash';
+import _, {orderBy} from 'lodash';
 import React from 'react';
-import WorkshopTableRow from './workshop_table_row';
-import {Table} from 'react-bootstrap';
+import {Button} from 'react-bootstrap';
+import {Table, sort} from 'reactabular';
+import color from '../../../../color';
+import SessionTimesList from './session_times_list';
+import FacilitatorsList from './facilitators_list';
+import WorkshopManagement from './workshop_management';
+import wrappedSortable from '../../../../templates/tables/wrapped_sortable';
```

Comment:
:dizzy_face: At this depth, it might be easier/cleaner to import by alias!
```javascript
import wrappedSortable from '@cdo/apps/templates/tables/wrapped_sortable';
```